traffic_ops/testing/api/v14/todb.go

```diff
@@ -275,6 +281,20 @@ INSERT INTO to_extension (name, version, info_url, isactive, script_file, server
 	return nil
 }
 
+// SetupServerCapabilities sets up seed server capabilities
```

Comment:
The seeding of server_capabilities is not needed

Reply:
I think this was a merge issue, they must have been removed from master after I submitted my PR